### ISSUE TITLE
App should listen to user continuously for help words

### ISSUE THREAD

**Issue by souvikbasu** [SQUATSFITNESS/safe-india-initiative-android#1]
Given that I am running the app
When I say words like "help" or "bachao"
Then the app should log my request for help in the database
And app should send notification to all nearby users for help

**Comment by DeepakG19** [comment]
This is going to be a battery intensive task. since you are performing a kind of NLP 24x7 on the Android platform. Possible solution is to identify a trigger point to initiate. For instance.. google's voice app

**Comment by souvikbasu** [comment]
Yes, it may be battery intensive. Our first approach will be to get a working solution. Once that is achieved we will look for a better solution which is not resource intensive. Can you try with Google voice api and get a working POC? We can fine tune it if it works
